public/src/js/models/events.ts

```diff
@@ -47,6 +51,7 @@ export class Event implements IEvent {
       this.limit = -1;
       this.generated_matches = false;
       this.host = hostIdOrKey;
+      this.event_host = 'Default Event Host';
```

Comment:
when we create the event, we know the host name as well, i think user.displayName has it. perhaps use that?

Reply:
done!